website/content/en/docs/best-practices/observability-best-practices.md

```diff
@@ -65,6 +65,19 @@ The most common types are:
  - `Gauge` Value can be increased and decreased as needed.
 5. `_total` suffix should be used for accumulating count. If your metrics has labels with high cardinality, like `pod`/`container` it usually means that you can aggregate it more, thus it will not require `_total` suffix.
 
+#### Recommended Metrics
+
+* `operator_health_status` - The metric reports the operator health status. It should be reported by the operator itself based on the internal operator validations.
+
+  This metric allows the cluster users to view all operators health with a single metric and improves the overall cluster observability.  
+  Valid health status values are:
```

Comment:
@sradco I found the comment you made [here](https://github.com/operator-framework/operator-sdk/pull/6315/files#r1107769855) useful. It would be nice to mention the reasoning behind having them as numbers in the doc too. I would suggest something like:

```
We would suggest to use numbers to represent the statuses in the values, since string values like: Healthy/Warning/Error, can be implemented in many ways that would cause unexpected consistency issues and disagreements, like spelling errors, capital letters etc.

One of the suggested set of values are:
* `0` - Indicates that the operator is healthy and working as expected.
* `1` - Indicates that the operator has some issues that needs to be addressed and can lead to loss of functionality.
* `2` - Indicates that the operator is unhealthy and there is a loss of functionality that should be addressed.

....

The set of values and their definitions should be clearly documented by the operator authors for any external observer or tool to infer them correctly or take necessary action accordingly. 
```

Reply:
Update. Please review again.